ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -1829,6 +1829,39 @@ let pool_reset_telemetry_uuid _printer rpc session_id params =
   let pool = get_pool_with_default rpc session_id params "uuid" in
   Client.Pool.reset_telemetry_uuid ~rpc ~session_id ~self:pool
 
+let pool_configure_update_sync _printer rpc session_id params =
+  let pool = get_pool_with_default rpc session_id params "uuid" in
+  let frequency =
+    Record_util.update_sync_frequency_of_string
+      (List.assoc "update-sync-frequency" params)
+  in
+  let day = List.assoc "update-sync-day" params in
+  let day_int =
+    try Int64.of_string day
+    with _ ->
+      failwith
+        "Failed to parse parameter 'update-sync-day': expecting an integer"
+  in
+  ( match (frequency, day_int) with
+  | `daily, d when d <> 0L ->
+      failwith
```

Comment:
Why is there a `\n` in these strings? And `failwith` is a very generic exception - should this not be an API error? Or should this never happen?